db/database.go

```diff
@@ -417,14 +423,18 @@ func (db *Database) DocCount() int {
 }
 
 func installViews(bucket base.Bucket) error {
+
+	syncData := "doc._sync"
```

Comment:
This is going to get more attention with the rest of the view support - I'll add more detail with that PR.